docs/aws-iam-policies/README.md

```diff
@@ -1,17 +1,24 @@
 
-The directory contains files that correspond to IAM policies in the
-prod and community Fedora AWS accounts.
+This directory contains files that correspond to IAM policies in the
+community Fedora AWS account. The policies for the prod Fedora AWS
+account are stored in the [infra ansible repo](https://pagure.io/fedora-infra/ansible).
 
 #### Prod Account:
 
 - `fcos-builds-bot` user has the following policies attached
-    - [fcos-upload-amis](prod-account/fcos-upload-amis)
-    - [fcos-poc-artifacts](prod-account/fcos-poc-artifacts)
+    - [fcos-upload-amis.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-upload-amis.json)
+        - Allows uploading/creating AMIs
+    - [fcos-poc-artifacts.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-poc-artifacts.json)
         - Allows writing to `fcos-builds` bucket
 
 #### Community Account
 
 - The `prod-account-match-fcos-builds-bot` group has the following policies attached:
-    - [prod-account-match-fcos-upload-amis](community-account/prod-account-match-fcos-upload-amis)
-    - [prod-account-match-fcos-poc-artifacts](community-account/prod-account-match-fcos-poc-artifacts)
+    - [prod-account-match-fcos-upload-amis](community-account/prod-account-match-fcos-upload-amis.json)
+        - Allows uploading/creating AMIs
+        - Compare to prod with
+            - `vimdiff https://pagure.io/fedora-infra/ansible/raw/main/f/files/aws/iam/policies/fcos-upload-amis.json community-account/prod-account-match-fcos-upload-amis.json`
```

Comment:
Neat! TIL vim supports automatically fetching remote URLs to a tempdir. And it correctly deletes it on exit. That's definitely useful.

Reply:
quite handy, right?